### ISSUE TITLE
Can I migrate from discourse to NodeBB using this tool?

### ISSUE THREAD

**Issue by TerrificMao** [akhoury/nodebb-plugin-import#228]
Can I migrate from discourse to NodeBB using this tool?

**Comment by akhoury** [comment]
Yes! when you get to the part where you select an exporter, select [`nodebb-plugin-import-discourse`](https://github.com/BenLubar/nodebb-plugin-import-discourse)

Let me know if you run into any issues, I will help you through it

**Comment by akhoury** [comment]
Keep in mind that the discourse exporter is over a year old, so we might have to make changes to it if you're exporting a fairly new discourse instance with many schema changes, but that shouldn't be that bad

**Comment by TerrificMao** [comment]
Hi akhoury, thanks so much for your quickly reply, really appreciate!!
I have installed the discourse with container deployment, and I also installed the the plugin that you suggested (nodebb-plugin-import-discourse) on another AWS EC2 ubuntu instance, but can I have some README or instructions? How can I migrate the database or the forum content from discourse to NodeBB?
Could you help give me some suggestions or any kindly help? Thanks in advance!!

**Comment by TerrificMao** [comment]
Hi akhoury, thanks so much for your quickly reply, really appreciate!!
I have installed the discourse with container deployment, and I also installed the the plugin that you suggested (nodebb-plugin-import-discourse) on another AWS EC2 ubuntu instance, but can I have some simple README or instructions? How can I migrate the database (pgsql in container discourse) or the forum content from discourse to NodeBB?
Could you help give me some suggestions or any kindly help? Thanks in advance!!


At 2019-04-30 20:29:13, "Aziz Khoury" <notifications@github.com> wrote:


Yes! when you get to the part where you select an exporter, select nodebb-plugin-import-discourse

Let me know if you run into any issues, I will help you through it

**Comment by Zerefdev** [comment]
> * go to your nodebb instance admin page, Extend > Plugins, find and install "nodebb-plugin-import" 
Not found! I managed to install it using npm
> ..then activate it then restart your nodebb instance
DONE

> * then visit the Plugins > Import page..
Not Found

**Comment by akhoury** [comment]
hmm.. 

1) any errors in the logs? 
2) what is the nodebb version you are using? the import plugin only supports 1 version at a time of nodebb https://github.com/akhoury/nodebb-plugin-import#this-plugin-only-supports
